result/arrow-spacing.js

```diff
@@ -0,0 +1,46 @@
+/*eslint-env es6*/
+
+// { "before": true, "after": true }
+(a) => {}
+
+// { "before": false, "after": false }
+(a)=>{}
+/*eslint arrow-spacing: "error"*/
+/*eslint-env es6*/
+
+()=> {};
+() =>{};
+(a)=> {};
+(a) =>{};
+a =>a;
+a=> a;
+()=> {'\n'};
+() =>{'\n'};
+/*eslint arrow-spacing: "error"*/
+/*eslint-env es6*/
+
+() => {};
+(a) => {};
+a => a;
+() => {'\n'};
+/*eslint arrow-spacing: ["error", { "before": false, "after": false }]*/
+/*eslint-env es6*/
+
+()=>{};
+(a)=>{};
+a=>a;
+()=>{'\n'};
+/*eslint arrow-spacing: ["error", { "before": true, "after": false }]*/
+/*eslint-env es6*/
+
+() =>{};
+(a) =>{};
```

Comment:
__[development][eslint]__
* Error - 'a' is defined but never used (no-unused-vars)
* Error - Unexpected space before => (arrow-spacing)
* Error - Missing space after => (arrow-spacing)